tests/extmod/vfs_open_error.py.exp

```diff
@@ -0,0 +1 @@
+OSError
```

Comment:
This exp file is either missing a corresponding test, or should just be removed.

Reply:
Thanks, I'll remove it, I merged the new test into (I hope) an appropriate existing test.

Reply:
Yes, the new execfile test is good.